inlong-common/src/main/java/org/apache/inlong/common/enums/DataProxyErrCode.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.common.enums;
+
+public enum DataProxyErrCode {
+
+    ERR_CODE_SUCCESS(0, "Ok"),
```

Comment:
Personally, I don't think it is necessary to add the `ERR_CODE` prefix, because the enumeration class name already indicates that all its instances are about ERROR CODE.

Reply:
This writing is just the name of the enumeration, and it is only a numerical value when it is interacted, and only a habit problem